docs/developer/community/contributing.mdx

```diff
@@ -266,9 +266,17 @@ Use relative imports.
 
 Try to combine multiple migrations into one, but remember not to mix changes on the database with updating rows in migrations. In other words, operations that alter tables and use `RunPython` to run methods on existing data should be in separate files.
 
+### Handling migrations between versions
+
+If you need to add the migration to the module that has different migrations on different versions,
+you need to start from the lower version (let's say 3.1) and create there the changes that you want to apply.
+Then you need to move to the upper version (in our case it will be 3.2), cherry-pick the newly added
+migration (on the 3.1 version), and create a migration merge if needed with use of
+`./manage.py makemigrations --merge`. Follow these steps on all versions and on the main branch.
```

Comment:
```suggestion
If you need to add the migration to the module that has different migrations on different versions,
you need to start from the lower version (let's say 3.1) and create there the changes that you want to apply.

Then you need to move to the upper version (in our case it will be 3.2), cherry-pick the newly added
migration (on the 3.1 version), and create a migration merge if needed with the use of
`./manage.py makemigrations --merge`. Follow these steps on all versions and on the main branch.
```